nixpkgs/pinned.nix

```diff
@@ -0,0 +1,6 @@
+import ((import <nixpkgs> {}).fetchFromGitHub {
```

Comment:
We should just use `builtins.fetchGit` instead to get rid of this impurity:

```import (builtins.fetchGit {
  url = https://github.com/NixOS/nixpkgs.git;
  rev = "5272327b81ed355bbed5659b8d303cf2979b6953";
  sha256 = "0182ys095dfx02vl2a20j1hz92dx3mfgz2a6fhn31bqlp1wa8hlq";
}) {}

Reply:
I removed the `{}` at the end because it's customary to allow the caller to pass in arguments and it better matches the `import <nixpkgs> {}` idiom when you switch to `import ./mypin.nix {}`